mlflow/store/artifact/azure_data_lake_artifact_repo.py

```diff
@@ -129,3 +140,103 @@ def _download_file(self, remote_file_path, local_path):
 
     def delete_artifacts(self, artifact_path=None):
         raise NotImplementedError("This artifact repository does not support deleting artifacts")
+
+    def _upload_to_cloud(self, cloud_credential_info, src_file_path, artifact_file_path):
+        if (
+            os.path.getsize(src_file_path) > _MULTIPART_UPLOAD_CHUNK_SIZE
+            and MLFLOW_ENABLE_MULTIPART_UPLOAD.get()
```

Comment:
```suggestion
            MLFLOW_ENABLE_MULTIPART_UPLOAD.get()
            and os.path.getsize(src_file_path) > _MULTIPART_UPLOAD_CHUNK_SIZE
```

can we swap the condtisions?